lib/create-client.js

```diff
@@ -16,16 +16,27 @@ auth.password = process.env.GH_PASSWORD || auth.password;
 console.assert(auth.username || auth.token);
 console.assert(auth.password || auth.token);
 
-var github = new GitHubApi({
+var github_params = {
   // required
   version: "3.0.0",
   // optional
   protocol: "https",
 //timeout: 5000,
-
   cachedb: '_cache.db',
   validateCache: false,
-});
+}
+
+ghe_host = auth.enterprise_host
+
+if (ghe_host) {
+  console.info("Connecting to GitHub Enterprise (" + ghe_host + ")..")
+  github_params.host = ghe_host;
+  github_params.pathPrefix = "/api/v3"
```

Comment:
This is part of the company specific URL and admins are actually free to use just about whatever root URL they like, so this is pretty specific to one company's GHE deployment.